frb_rust_bindgen_io/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "flutter_rust_bridge_bindgen_io"
```

Comment:
uber-nit: "bindgen io" looks very specific to the underlying tech it uses. For example, maybe one day there is a lib called awesome_new_bindgen, then we have to rename this crate. So shall we name it more general (well naming is hard ;) )

Reply:
Sooo I agree naming is difficult and I could change to any other name, no problem! It's just that we don't need to publish this crate, it's a simple internal tool to regenerate allo_isolate bindings. It's as you wish really!

Will add some details to the reason of this crate

Reply:
If it durably only treats allo_isolate, it could be called `allo_isolate_bindgen` actually :sweat_smile:

Reply:
This crate might more or less be dedicated to `frb_dart` bindings generation so...

I felt like it would a better practice to keep crates on the same flat level but here this one is clearly a requirement of `frb_dart`, so I'll look into nesting it in `frb_dart` directly. Simpler, clearer, not impacting workspace.

I see you suggesting having common code generation functions though :) I just feel that they already exist in `frb_codegen` which could be used as a library. This seems a bit out of the scope of this PR isn't it?

Reply:
> they already exist in frb_codegen which could be used as a librar

* frb_codegen: users use it at/before compile time to generate user code
* frb_dart/frb_rust: user use it at runtime
* this_new_crate: *we* use it at/before compile time, to generate *code in frb_dart/frb_rust*, i.e. generate code *inside the flutter_rust_bridge* repo

so they are somehow quite different ;)

Reply:
> This seems a bit out of the scope of this PR isn't it?

No, it is just a naming problem (change a few characters) ;)

IMHO we can put everything related to "generating code in frb_dart/frb_rust" inside this new crate, that's the main point why I want it to have a more general name

Reply:
>    - frb_codegen: users use it at/before compile time to generate user code
>    - frb_dart/frb_rust: user use it at runtime
>    - this_new_crate: we use it at/before compile time, to generate code in frb_dart/frb_rust, i.e. generate code inside the flutter_rust_bridge repo

Sums it up really well! :ok_hand: 

Not sure if you noticed but I moved this crate _inside_ `frb_dart` in order to avoid confusion with the rest of the code base and because it shares configuration with `frb_dart` for simplicity and maintainability. I kept the "bindgen" naming for now, because I think it is plain bingen of an external crate. Shall I rename this folder to "source_generator"? Seems ok to make it live in `frb_dart` (unpublished of course)